specs/0116-LLM-assisted-translations.md

```diff
@@ -0,0 +1,51 @@
+- Feature Name: LLM-Assisted Translations in Pontoon
+- Created: 2024-01-12
+- Associated Issue: #3064
+
+# Summary
+
+Introduce LLM-assisted translations in Pontoon to provide alternative, formal, and informal translation options, enhancing translation quality.
+
+# Motivation
+
+Modern Large Language Models (LLMs) — like OpenAI’s GPT-4 — have reached a quality level for localization tasks that is on par with dedicated machine translation engines. This feature is primarily focused on introducing a tool that aims to improve translation quality in Mozilla's projects. By incorporating LLM-assisted translations, Mozilla not only remains competitive with other Translation Management Systems (TMS) providers but also embraces new technology to enhance the overall translation process. This feature also lays the groundwork for future integrations that could further enhance efficiency and broaden the scope of translation improvements within Mozilla's localization efforts.
```

Comment:
```suggestion
Modern Large Language Models (LLMs) — like OpenAI’s GPT-4 — have reached a quality level for localization tasks that is on par with dedicated machine translation engines. This feature is primarily focused on introducing a tool that aims to improve translation quality in Mozilla's projects. By incorporating LLM-assisted translations, Mozilla not only remains competitive with other Translation Management Systems (TMS) but also embraces new technologies to enhance the overall translation process. This feature also lays the groundwork for future integrations that could further enhance efficiency and broaden the scope of translation improvements within Mozilla's localization efforts.
```

Reply:
Why not leave "providers" in? Mozilla is a TMS provider, not TMS.

Reply:
Good point, I didn't think of the subject of the sentence.